include/llvm/MC/SectionKind.h

```diff
@@ -31,10 +31,9 @@ class SectionKind {
            /// ExecuteOnly, Text section that is not readable.
            ExecuteOnly,
 
-           /// RandAddr - Text section should go on a page which will be randomly
-           /// located in memory.
+           /// TextRand - Text section location in memory should be randomized
+           /// by the loader.
            TextRand,
```

Comment:
While we're bikeshedding names, is this a good name? It's the only place that "Rand" seems to still be used?